### DISCUSSION TITLE
Notion roadmap

### DISCUSSION THREAD

**Discussion by rootulp** [celestiaorg/celestia-app#668] [General]
We have a Notion [roadmap](https://www.notion.so/celestiaorg/2cb2e943161a487d8bdd64f908103b41?v=9824ff2514c04112a2b71b35ec7a14b9) that describes a timeline of when core/app expects to work on features. The Notion doc is meant to communicate our progress to other internal teams.

This discussion is to gather feedback on the process. In particular:
- Is Notion the right medium for the roadmap?
- Is there any info that internal teams would benefit from that isn't captured in the Notion?
- Would it be helpful for external contributors to see the Notion?

cc: @evan-forbes

**Comment by evan-forbes** [comment]
we can start using github's new gant style roadmaps! 

I think this would be particularly useful in the future when people are waiting for specific features/fixes etc.

**Reply by rootulp** [reply]
+1 providing external visibility into our roadmap seems beneficial
